src/main/kotlin/racingcar/domain/car/Car.kt

```diff
@@ -0,0 +1,16 @@
+package racingcar.domain.car
+
+import racingcar.domain.car.action.MovingStrategy
+import racingcar.domain.car.status.Name
+import racingcar.domain.car.status.Position
+
+data class Car(private val carName: String) {
+    val name: Name = Name(carName)
+    var position: Position = Position(0)
```

Comment:
```suggestion
    var position: Position = Position(0)
        private set 
```
변경 가능한 var 변수는 외부에서의 무분별한 수정을 막기 위해 private setter 를 선언하는것도 좋은 방법입니다. 😄

Reply:
조언 감사합니다! 어제 온라인 강의때도 해당 주제에 대해서 얘기가 나왔었는데요. 추후에는 외부에서 접근 가능한 `var` 프로퍼티의 경우 `private setter` 선언을 고려해 보도록 하겠습니다. 😃 

- https://github.com/next-step/kotlin-racingcar/commit/cabcee7ba666aab7131cf67f19573295920fe05a